grpcurl_test.go

```diff
@@ -260,6 +260,20 @@ const (
   "payload": {
     "body": "SXQncyBCdXNpbmVzcyBUaW1l"
   }
+}`
+	fullResponse1 = `{
+  "payload": {
+    "type": "COMPRESSABLE",
+    "body": "SXQncyBCdXNpbmVzcyBUaW1l"
+  },
+  "username": "",
+  "oauthScope": ""
+}`
+	response1 = `{
```

Comment:
Why retain this constant? I see it used for half-duplex stream test. Why does that test behave differently than the unary test?

Reply:
> Why does that test behave differently than the unary test?

Heh, to be quite honest, I have no idea, and it was surprising me as well. Can you tell?

Reply:
Anyways, this will probably go away with the re-work to make this a flag.